lib/model/movie_data_model.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+
+class MovieDataModel {
+  bool? adult;
+  String? backdropPath;
+  List<num>? genreIds;
+  num? id;
+  String? originalLanguage;
+  String? originalTitle;
+  String? overview;
+  num? popularity;
+  String? posterPath;
+  String? releaseDate;
+  String? title;
+  bool? video;
+  num? voteAverage;
+  num? voteCount;
```

Comment:
Beware of marking **all** of those parameters as nullables, as you'll have to either force-unwrap them (terrible bad idea) or provide default values.
Better look into the TheMovieDB API documentation on which ones are nullables and which others aren't!